src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/Attendance.java

```diff
@@ -75,7 +78,7 @@
     @Column(name = "MODIFY_USER_ID")
     private String modifyUserId;
 
-    public Attendance(Long id, @NotNull Long bookingId, @NotNull Long eventId, @NotNull Long eventLocationId, @NotNull LocalDate eventDate, @NotNull TimePeriod period, @NotNull @Size(max = 6) String prisonId, Boolean paid, Boolean attended, AbsentReason absentReason, @Size(max = 240) String comments, Long caseNoteId, LocalDateTime createDateTime, String createUserId, LocalDateTime modifyDateTime, String modifyUserId) {
+    public Attendance(Long id, @NotNull Long bookingId, @NotNull Long eventId, @NotNull Long eventLocationId, @NotNull LocalDate eventDate, @NotNull TimePeriod period, @NotNull @Size(max = 6) String prisonId, Boolean paid, Boolean attended, AbsentReason absentReason, AbsentSubReason absentSubReason, @Size(max = 240) String comments, Long caseNoteId, LocalDateTime createDateTime, String createUserId, LocalDateTime modifyDateTime, String modifyUserId) {
```

Comment:
all this has been delomboked since there is a mix of kotlin and java in this repo